### PR TITLE
Master 1.0.0

### PR DESCRIPTION
**Pull request by dontforgit**

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+            echo "<pre>", print_r($vVariableToOutput), "</pre>";
+        } else {
+            var_dump($vVariableToOutput);
+        }
+
+        if ($bDieAfterOutput !== false) {
+            die();
+        }
+    }
+}
```
Comment (dontforgit) [inline index.php]: You need a new line at the EOF.

File: index.php [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+ */
+if (function_exists('echo_pre') === false) {
+    function echo_pre($vVariableToOutput, $bDieAfterOutput = false)
+    {
+        if (is_object($vVariableToOutput) === true || is_array($vVariableToOutput) === true) {
+            echo "<pre>", print_r($vVariableToOutput), "</pre>";
+        } else {
+            var_dump($vVariableToOutput);
+        }
+
```
Comment (dontforgit) [inline index.php]: Remove this space.

File: index.php (hunk 1 above)
Comment (summersjwfu) [inline index.php]: What he said

Comment (dontforgit) [reply index.php]: I will

**Pull request merged**